tasks/docker.yml

```diff
@@ -1,9 +1,73 @@
 ---
-- name: Installing docker
+- name: Installing Python docker module requirements
   become: "{{ not lookup('env','VIRTUAL_ENV') }}"
   pip:
     name: docker
   register: my_result
   until: my_result is succeeded
   retries: 5
   delay: 10
+  environment:
+    use_proxy: "{{ use_proxy }}"
+    http_proxy: "{{ http_proxy if use_proxy else '' }}"
+    https_proxy: "{{ https_proxy if use_proxy else '' }}"
+
+- name: populate service ansible_facts
+  service_facts:
+
+- name: Ensure group "docker" exists
+  become: yes
+  group:
+    name: docker
+    state: present
```

Comment:
A couple of when deploy_base_infra are missing, right?

Reply:
not needed, as it's already specified in the "include" statement in `tasks/main.yml` ;-)